packages/forms-web-app/src/views/interested-party-guide/have-your-say-examination.njk

```diff
@@ -0,0 +1,99 @@
+{% extends "layouts/layout.njk" %}
+
+{% block pageTitle %}
+  Have your say during the examination of the application
```

Comment:
This should be 'Have your say during the Examination of the application - National Infrastructure Planning'

Reply:
Fixed